pkgs/development/python-modules/pymee/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  aiohttp,
+  websockets
+}:
+
+buildPythonPackage rec {
+  pname = "pymee";
+  version = "2.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "FreshlyBrewedCode";
+    repo = "pymee";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-4XKd0lZ6RAsG2zXjKMUeST6cNcg+SjT371gxLIhxkAA=";
+  };
+
+  build-system = [ setuptools ];
+  dependencies = [ aiohttp websockets ];
+
+  pythonImportsCheck = [ "pymee" ];
+
```

```suggestion

  # no tests
  doCheck = false;

```
Please disable tests explicitly.

---

why do I always forget something